exercise 1-semantic and css version.html

```diff
@@ -49,8 +52,7 @@ <h2>Test this page</h2>
                     </li>
                 </ul>
             </main>
-    
-            <aside style="width: 20%; background-color: lightBlue; margin: 0.3em; padding: 15px;">
+            <aside class="side-content">
                 <h2>HTML</h2>
```

Comment:
Those <p> tags(HTML, CSS, URL) content do not coincide with the content of Nelson's example, but is not a big deal for me... what do you think?

Reply:
also... In my opinion theres two articles inside the section and you wrote only one, what do you think about having two articles dividing "about this page" and "test this page"

Reply:
About the description, i dont think it's a big deal its just a lore ipsum, however i would add the article tag on the "test this page" section